mime.types

```diff
@@ -0,0 +1,55 @@
+types {
```

Comment:
Is there any reason these are grouped the way they are? i.e., text, image, application, text, image, application, audio, video. Any reason not to alphabetize them, either by type or by extension?

Reply:
Also, do we need
  image/svg+xml                              svg;
  font/otf                              otf;
  font/ttf                              ttf;
  font/woff                              woff;
  font/woff2                              woff2;

Do we need to define our .fec extension?

Reply:
@rfultz thanks for your comments! I originally borrowed from the nginx documentation (https://www.nginx.com/resources/wiki/start/topics/examples/full/) and added spaces for more readability. After testing together, we determined that order doesn't seem to matter, so we can alphabetize/group the types.

However, after further research the example `mime.types` file in the nginx buildpack repo is more comprehensive, so I'm going to go with that one (alphabetized and grouped): https://github.com/cloudfoundry/nginx-buildpack/blob/master/fixtures/mainline/mime.types

Reply:
Noting that we don't need to set a mime type for .fec files, since those are served by the docquery server: 

```
curl "https://docquery.fec.gov/dcdev/posted/1490529.fec" -I
...
Content-Type: mime.types:application/fecprn
```